ui/apps/platform/src/Containers/NetworkGraph/types/flow.type.ts

```diff
@@ -1,5 +1,11 @@
+import { L4Protocol } from 'types/networkFlow.proto';
```

Comment:
Funky identifier reminded me of this old sci-fi club,
https://en.wikipedia.org/wiki/L5_Society

which reminded me of what one of the original astronauts complained about a few years ago,
> You Promised Me Mars Colonies. Instead, I Got Facebook.

Reply:
😂